RecoTauTag/RecoTau/python/tools/runTauIdMVA.py

```diff
@@ -58,6 +88,7 @@ def __init__(self, process, debug = False,
             if discr not in TauIDEmbedder.availableDiscriminators:
                 raise RuntimeError('TauIDEmbedder: discriminator "{}" is not supported'.format(discr))
         self.toKeep = toKeep
+        self.working_points_v2p5 = deepcopy(WORKING_POINTS_v2p5)
```

Comment:
Maybe I am being too much cautious here - the idea was to not pass the original WP dictionary and keep it like a true constant, because maybe it can be changed in the downstream functions (because of mutability) and this is not good. But I was a bit reluctant to check what is happening there. Should I remove it?